.gitignore

```diff
@@ -42,3 +42,5 @@ local.properties
 # Sandbox stuff
 _sandbox
 
+signing.properties
+droidconit.jks
```

Comment:
I was about to say...  That's weird. @tiwiz please don't add files that aren't going to be published to the git ignore by name.